lib/msal-browser/package.json

```diff
@@ -55,7 +55,7 @@
     "test:coverage:only": "npm run clean:coverage && npm run test:coverage",
     "build:all": "npm run build:common && npm run build",
     "build:common": "cd ../msal-common && npm run build",
-    "build:modules": "rollup -c",
+    "build:modules": "rollup -c --strictDeprecations --bundleConfigAsCjs",
```

Comment:
why common-js bundling?

Reply:
Just avoiding making more invasive changes to the current configuration: https://rollupjs.org/command-line-interface/#bundleconfigascjs